OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/FirebaseAnalytics.java

```diff
@@ -809,4 +811,22 @@ public void trackProfileScreenEvent(@NonNull String eventName, @NonNull String b
         final FirebaseEvent event = new FirebaseEvent(eventName, biValue);
         logFirebaseEvent(event.getName(), event.getBundle());
     }
+
+    @Override
+    public void trackVideoDownloadQualityClicked(String eventName, String biValue, String courseId) {
+        final FirebaseEvent event = new FirebaseEvent(eventName, biValue);
+        if (!TextUtils.isEmpty(courseId)) {
+            event.putCourseId(courseId);
+        }
+        logFirebaseEvent(event.getName(), event.getBundle());
+    }
+
+    @Override
+    public void trackVideoDownloadQualityChanged(@NonNull @NotNull VideoQuality selectedVideoQuality, @NonNull @NotNull VideoQuality oldVideoQuality) {
+        final FirebaseEvent event = new FirebaseEvent(Analytics.Screens.VIDEO_DOWNLOAD_QUALITY_CHANGED,
```

Comment:
Can we use Analytics.Event value instead of Analytics.Screen.

Reply:
done.